program-runtime/src/compute_budget.rs

```diff
@@ -743,15 +667,12 @@ mod tests {
                 compute_unit_limit: DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT as u64 * 7,
                 ..ComputeBudget::default()
             },
-            false,
             false
```

Comment:
I think the comment above is also incorrect:
```rust
// assert normal results when not requesting heap frame when the feature is disabled
```

since the feature is always enabled

Reply:
right, this whole test is obsoleted - the remaining test is already covered in `test_process_instructions`. I have removed it.